sql/moz-fx-data-shared-prod/mozilla_org_derived/blogs_goals_v2/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: Blogs Goals
+description: |-
+  Please provide a description for the query
+owners:
+- example@mozilla.com
+labels:
+  incremental: true
+  owner1: example
+scheduling:
+  dag_name: bqetl_default
+bigquery:
+  time_partitioning:
+    type: day
+    field: ''
```

Comment:
I think it's partitioned on date

Reply:
May want to have it match https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-marketing-prod/ga_derived/blogs_goals_v2/metadata.yaml